index.md

```diff
@@ -9,20 +9,22 @@ layout: default
 A national, distributed computing partnership for data-intensive research
 
 <div id="osg-special-banner">
-  <p class="special-banner-1"><strong>Extra OSG Support for COVID-19 Research</strong></p>
+  <p class="special-banner-1"><strong>We are here to help with your <a href="https://www.nsf.gov/pubs/2021/nsf21528/nsf21528.htm/">Campus Cyberinfrastructure (CC*) Proposal!</a> Please contact us at <a href="mailto:cc-star-proposals@opensciencegrid.org">cc-star-proposals@opensciencegrid.org</a></strong></p>
```

Comment:
That is a LOT of strong (i.e., bold) text. Looking at last time, we bolded just the phrase “NSF Campus Cyberinfrastructure (CC*) Campus Computing and the Computing Continuum”.